src/blog/2024/10/exploring-flowfuse-sbom-feature.md

```diff
@@ -0,0 +1,72 @@
+
+1. Navigate to your Node-RED application within the FlowFuse platform.
+
+![Image showing the 'Applications' option in the FlowFuse platform](./images/applications-options-in-the-ff.png){data-zoomable}
+_Image showing the 'Applications' option in the FlowFuse platform._
+
+2. Click the **Dependencies** option at the top to switch to the SBOM interface.
+
+![Image showing the 'Dependencies' option in the FlowFuse platform for the SBOM interface.](./images/dependencies-tab-option.png){data-zoomable}
+_Image showing the 'Dependencies' option in the FlowFuse platform for the SBOM interface._
+
+*Note: This feature is only available for FlowFuse Enterprise self-hosted and Enterprise Tier users.*
+
+### Understanding What the SBOM Interface Shows
+
+Once you navigate the tab, you will see a list of all the packages installed within your Node-RED Cloud instances and devices associated with that application. This includes the package names and versions, the number of devices and instances using each version, and additional details such as the latest available version of each package and the time since its release.
+
+![Image showing the Dependencies tab along with the detailed notes of each item displayed.](./images/the-dependency-tab-info.png){data-zoomable}
+_Image showing the Dependencies tab along with the detailed notes of each item displayed._
+
+### Now What? Make the Most of SBOM
+
+Now that you've explored the SBOM interface, you can use it to enhance the security of your Node-RED applications:
+
+1. **Monitor Regularly**: Check your SBOM often to identify outdated or vulnerable packages. Staying updated can prevent potential security threats.
+
+2. **Upgrade Packages**: If you find any packages flagged for vulnerabilities, upgrade them promptly. This will help secure your application and improve performance.
+
+3. **Assess Third-Party Nodes**: Review the third-party nodes you use. If any lack regular updates, consider alternatives to ensure your application remains secure and well-maintained.
+
+Adopting these practices in your development routine will enhance your applications' security and cultivate a proactive maintenance culture. Security is a team effort, and staying vigilant about your dependencies is essential for protecting your projects. The FlowFuse SBOM feature is an invaluable auditing tool, accessible to all team members with the appropriate permissions, allowing everyone to contribute to maintaining security seamlessly.
+
+**[Sign up](https://app.flowfuse.com/) for FlowFuse now and enjoy a free trial to explore all our features! Discover how you can enhance your Node-RED projects and accelerate your production processes.**
+
+*If you are an educator or student, we offer support and provide free licenses. Contact us for more information [here](/education/).*
+
+### Conclusion
```

Comment:
Since this is not a very long article and there is nothing too much you are saying new in the summary you can skip the conclusion section and end with the call to action above.